docs/cli/tkn-results_taskrun_describe.md

```diff
@@ -0,0 +1,54 @@
+## tkn-results taskrun describe
+
+Describe a TaskRun
+
+### Synopsis
+
+Describe a TaskRun by name or UID. If --uid is provided, then TaskRun name is optional.
+
+```
+tkn-results taskrun describe [taskrun-name]
+```
+
+### Examples
+
+```
+Describe a TaskRun in namespace 'foo':
+    tkn-results taskrun describe my-taskrun -n foo
+
+Describe a TaskRun in the current namespace
+    tkn-results taskrun describe my-taskrun
+
+```
+
+### Options
+
+```
+  -A, --all-namespaces   use all namespaces
```

Comment:
That seems redundant to me. It's good for list(ing), but not for getting/describing a particular taskrun.

Reply:
Yes, I think that makes sense.
I also looked at the other k8s-related CLIs but I couldn't find the option for --all-namespaces.
Will do the changes. I think this change will go for logs command as well. I will remove it